tests/integration/targets/git_config/files/gitconfig

```diff
@@ -3,4 +3,9 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 
 [http]
-        proxy = foo
+proxy = foo
```

Comment:
Can you restore the indent? My guess is that it was intentional.

Reply:
Reverted